tests/lib/nested.sh

```diff
@@ -49,25 +49,54 @@ get_qemu_for_nested_vm(){
     command -v qemu-system-x86_64
 }
 
-get_image_url_for_nested_vm(){
+get_google_image_url_for_nested_vm(){
     case "$SPREAD_SYSTEM" in
-    ubuntu-16.04-64)
-        echo "https://cloud-images.ubuntu.com/xenial/current/xenial-server-cloudimg-amd64-disk1.img"
-        ;;
-    ubuntu-18.04-64)
-        echo "https://cloud-images.ubuntu.com/bionic/current/bionic-server-cloudimg-amd64.img"
-        ;;
-    ubuntu-19.10-64)
-        echo "https://cloud-images.ubuntu.com/eoan/current/eoan-server-cloudimg-amd64.img"
-        ;;
-    ubuntu-20.04-64)
-        echo "https://cloud-images.ubuntu.com/focal/current/focal-server-cloudimg-amd64.img"
-        ;;
-    *)
-        echo "unsupported system"
-        exit 1
-        ;;
-    esac
+        ubuntu-16.04-64)
+            echo "https://storage.googleapis.com/spread-snapd-tests/images/xenial-server-cloudimg-amd64-disk1.img"
```

Comment:
What's the plan for refreshing these images in GC storage? Can we arrange to have them re-downloaded daily? It is important for preseeding tests to use relatively current images.

Reply:
No, this has to me manually downloaded, for that there is a cache system which I am going to integrate soon which allows that. 
Something that I can do is to create a spread cron which update those images monthly, does it work for you?

Reply:
Yes, a cron job sounds good, however a month is too long, I think we should refresh them every few days or even every day.

Reply:
ok, I'll create the cron job in that case

Reply:
Actually it's only important for recent releases (19.10, 20.04). Old ones can be downloaded once a month.